hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/acl/prefix/OMPrefixAclRequest.java

```diff
@@ -93,6 +94,11 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
 
       omPrefixInfo = omMetadataManager.getPrefixTable().get(prefixPath);
 
+      if(omPrefixInfo == null) {
+        throw new OMException("No prefix info for the prefix path: " + prefixPath,
```

Comment:
This line might cause checkstyle error. Please run `hadoop-ozone/dev-support/checks/checkstyle.sh`